frontend/src/components/ShootWorkers/MachineImage.vue

```diff
@@ -121,6 +121,17 @@ export default {
       }
       return join(hintText, ' / ')
     },
+    hintColor () {
+      if (this.machineImage.expirationDate ||
```

Comment:
Shouldn't we create a class for this classification, version, hint, ... stuff?

Reply:
Yes, would be a good idea. But not as part of this PR